spec/features/mfa_enrollability_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+describe 'TOTP enrollability' do
```

Comment:
Backup codes and PIV/CAC would also probably fit nicely in here, right?

Reply:
Indeed!